src/Rings/mpolyquo-localizations.jl

```diff
@@ -1600,7 +1600,14 @@ function simplify(L::MPolyLocRing{<:Any, <:Any, <:Any, <:Any, <:MPolyPowersOfEle
   return Lnew, hom(L, Lnew, f), hom(Lnew, L, finv, check=false)
 end
 
+# The `simplify` routine uses Singular's "elimpart". This does not work 
```

Comment:
I think this comment should be attached to the *next* method, as the method it documents right now does not call into Singular at all?